### PR TITLE
Fixing travis.yml file so it passes.

### PR DESCRIPTION
**Pull request by jmhossler**
our travis-ci pipeline was failing due to inability to start the emulator to run tests against. This has tests pass.

### CODE REVIEW

**Comment by jmhossler** [comment]
Example pass: https://travis-ci.org/jmhossler/roastd/builds/360068816

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```
Comment (jmhossler) [inline .travis.yml]: I went through so many iterations here.. I'm not sure if this is required but it works now

File: .travis.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ cache:
 env:
   global:
    - ANDROID_API_LEVEL=27
-   - EMULATOR_API_LEVEL=27
+   - EMULATOR_API_LEVEL=24
```
Comment (tlmurphy) [inline .travis.yml]: Would having the api level for the emulator at 24 cause any foreseeable issues?

Comment (jmhossler) [inline .travis.yml]: There are some issues with having emulators past 24 apparently. Seems to be an issue on googles end.

File: .travis.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ cache:
 env:
   global:
    - ANDROID_API_LEVEL=27
```
Comment (jmhossler) [inline .travis.yml]: was it android api 27 or 26 that we are using?

File: .travis.yml [hunk 4]
```diff
@@ -21,32 +21,34 @@ android:
...
-  - emulator -avd test -no-skin -no-audio -no-window &
+  - echo no | android create avd --force -n test -t android-$EMULATOR_API_LEVEL --abi default/armeabi-v7a
+  - emulator -avd test -no-skin -no-window &
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
 script:
         - ./gradlew clean
-        - ./gradlew assembleDebug
-        - ./gradlew assembleDebugAndroidTest
+        - ./gradlew build connectedCheck
```
Comment (jmhossler) [inline .travis.yml]: This runs the tests

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # roastd
 ![roastd](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSbvinN-LEe2yDoA-N4D-myW60qBrQKByHuMimjNttSwKDut5_x)
 
-[![Build Status](https://travis-ci.org/jmhossler/roastd.svg?branch=master)](https://travis-ci.org/jmhossler/roastd)
+[![Build Status](https://travis-ci.org/Roastd/roastd.svg?branch=master)](https://travis-ci.org/Roastd/roastd)
```
Comment (jmhossler) [inline README.md]: track new CI build

File: .travis.yml (hunk 3 above)
Comment (tlmurphy) [reply .travis.yml]: I'm using api 26 for my emulator because the api 27 emulator had weird issues on my end.

Comment (jmhossler) [reply .travis.yml]: Okay, I can lower that.

**Comment by jmhossler** [comment]
Waiting until the newest commits pass before merging

**Comment by jmhossler** [comment]
well, it failed.

**Pull request merged**